tests/statusHelper.ts

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2018 Lindorff Oy
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { Status } from "../lib/interfaces";
+export class StatusHelper {
+    public static toStatus(statusName: string, isDone?: boolean): Status {
```

Comment:
...on the other hand, this doesn't really need a class around it, so it would be entirely possible to just export this function alone.

Doing this, the `*.spec.ts` files wouldn't have to do the awkward `const toStatus = StatusHelper.toStatus` dance.